source/documentation/monitoring_apps/metrics.md

```diff
@@ -2,94 +2,56 @@
+- the [`paas-prometheus-exporter`](https://github.com/alphagov/paas-prometheus-exporter)
+- the [`paas-metric-exporter`](https://github.com/alphagov/paas-metric-exporter) to push metrics data in the [StatsD](https://github.com/etsy/statsd/wiki) [external link] format
 
 You can also view all metrics in a one-off snapshot by installing the Cloud Foundry CLI [log cache plug-in](https://github.com/cloudfoundry/log-cache-cli#installing-plugin) [external link].
 
-### Use the Prometheus endpoint
+### PaaS Prometheus exporter app
 
-Prometheus uses the following API endpoints to request metrics from Cloud Foundry:
+To use the PaaS Prometheus exporter app, you deploy it as an app on PaaS. The current metrics supported by this app are:
 
-- `https://metrics.cloud.service.gov.uk/metrics` for the Ireland region
-- `https://metrics.london.cloud.service.gov.uk/metrics` for the London region
-
-PaaS maintains these endpoints for free, so you can access all available metrics for free. You can configure Prometheus manually to filter out any metrics you do not need.
-
-You must set up Prometheus to request metrics from your region's API endpoint.
-
-1. [Install Prometheus](https://prometheus.io/docs/prometheus/latest/getting_started/) [external link].
-
-1. You must set up a bearer token so the API endpoint can authenticate your Prometheus request. We recommend you use a `bearer_token_file`. Set up an automated cron job to run the following command every 5 minutes:
-
-	```
-	cf oauth-token > /path/to/bearer_token_file.txt
-	```
-
-	where:
-	- `cf oauth-token` is the command that generates a bearer token
-	- `/path/to/bearer_token_file.txt` is the location and name of the `bearer_token_file` used by the Prometheus configuration
-
-1. Configure Prometheus to read the bearer token from the `bearer_token_file.txt`. Refer to the Prometheus [configuration documentation](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#ingress) [external link] for more information.
-
-#### Use Docker to run Prometheus locally
-
-You can set up Prometheus to request metrics from your region's API endpoint by using [Docker](https://www.docker.com/) [external link] to run a local instance of Prometheus.
-
-1. Save the following script as `test-metrics.sh`:
-
-	```
-	#!/usr/bin/env bash
-	set -ue
+- CPU
+- RAM
+- disk usage data
+- memory usage data
+- app crashes
+- app requests
+- app response times
 
-	echo "
-	global:
-	  scrape_interval: 1m
-	  evaluation_interval: 1m
-	  scrape_timeout: 1m
+Before you set up the PaaS Prometheus exporter app, you will need:
 
-	scrape_configs:
-	  - job_name: PaaS
-	    bearer_token: $(cf oauth-token | sed 's/bearer //')
-	    scheme: https
-	    static_configs:
-	      - targets:
-	        - API_ENDPOINT:443
-	" > prometheus.yml
+- a monitoring system to store the metrics with an accompanying metrics endpoint
```

Comment:
You can say:

```
A Prometheus service that would scrape and store the metrics and expose them
```

In the prometheus world, the prometheus world the prometheus service would "scrape" (that is, do regular queries to a `/metrics` endpoint to read the metrics.